src/GafferBindings/CompoundNumericPlugBinding.cpp

```diff
@@ -54,6 +56,23 @@ namespace
 {
 
 template<typename T>
+std::string maskedCompoundNumericPlugRepr( const T *plug, unsigned flagsMask )
+{
+	std::string extraArgs = "";
+
+	const IECore::GeometricData::Interpretation interpretation = plug->interpretation();
+	if( interpretation != IECore::GeometricData::None && interpretation != IECore::GeometricData::Color )
```

Comment:
I'm not necessarily in favour of setting it to None, but I would say that all our Color3f plugs should probably have the same interpretation.  So either we should set it to None here, or we should somehow set the default to be Color on Color3f plugs ( Matti pointed out that the latter might be difficult because Color3f is just a typedef ).